fcrepo-ldpath/README.md

```diff
@@ -14,24 +14,24 @@ Note: The LDPath service requires an LDCache backend, such as `fcrepo-service-ld
 
 ## Usage
 
-The LDPath service responds to `GET` and `POST` requests at URL locations that map to Fedora resources.
+The LDPath service responds to `GET` and `POST` requests using any accessible resources as a context.
 
-For example, if the `fcrepo.baseUrl` is set as `http://localhost:8080/fcrepo/rest`, then a request to
-`http://localhost:9086/ldpath/path/to/fedora/object` will be applied to the Fedora resource at
-`http://localhost:8080/fcrepo/rest/path/to/fedora/object`.
+For example, a request to
+`http://localhost:9086/ldpath/?context=http://localhost/rest/path/to/fedora/object`
+will apply the appropriate ldpath program to the specified resource. Note: it is possible to
+identify non-Fedora resources in the context parameter.
 
 A `GET` request can include a `ldpath` parameter, pointing to the URL location of an LDPath program:
 
-    `curl http://localhost:9086/ldpath/path/to/fedora/object?ldpath=http://example.org/ldpath`
+    `curl http://localhost:9086/ldpath/?context=http://localhost/rest/path/to/fedora/object?ldpath=http://example.org/ldpath`
```

Comment:
This does not appear to follow the standard HTTP query parameter syntax of:
```
http://host/path?param0=x&param1=y
```
Whereas here, you have:
```
http://host/path?param0=x?param1=y
```
I am just verifying that the `ldpath` service takes this different approach.